crypto/ml_dsa/ml_dsa_key.c

```diff
@@ -379,14 +400,11 @@ int ossl_ml_dsa_key_pairwise_check(const ML_DSA_KEY *key)
  * @brief Generate a public-private key pair from a seed.
  * See FIPS 204, Algorithm 6 ML-DSA.KeyGen_internal().
  *
- * @param seed The input seed
- * @param seed_len The size of entropy (Must be 32 bytes)
  * @param out The generated key (which contains params on input)
```

Comment:
which includes the seed

Reply:
The seed is no longer a separate argument, it set into the "prekey" by a separate call. What specific language would you like to see here?

Reply:
```suggestion
 * @param out The generated key (which contains params on input, including the seed)
```